astro.config.mjs

```diff
@@ -12,7 +12,7 @@ export default defineConfig({
       description:
         "Web Monetization allows websites to automatically receive payments from users, facilitated by the user agent and a user's preferred monetization provider.",
       customCss: [
-        './node_modules/@interledger/docs-design-system/src/styles/green-theme.css',
```

Comment:
Breaking change mentioned in release notes: https://github.com/interledger/docs-design-system/releases/tag/v0.3.0